frontend/src/component/Account/UserSpacesConfig.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import {
+  BtnSwitch,
+  IconButton,
+  ConfirmPopup,
+  ROLE_LIST,
+  TracimComponent,
+  TLM_ENTITY_TYPE as TLM_ET,
+  TLM_CORE_EVENT_TYPE as TLM_CET
+} from 'tracim_frontend_lib'
+
+import { newFlashMessage } from '../../action-creator.sync.js'
+import { deleteWorkspaceMember, getUserWorkspaceList, getWorkspaceMemberList } from '../../action-creator.async.js'
+
+export class UserSpacesConfig extends React.Component {
+  constructor (props) {
+    super()
+
+    this.state = {
+      workspaceList: [],
+      spaceBeingDeleted: null
+    }
+
+    props.registerLiveMessageHandlerList([
+      { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.CREATED, handler: this.handleMemberCreated },
+      { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.MODIFIED, handler: this.handleMemberModified },
+      { entityType: TLM_ET.SHAREDSPACE_MEMBER, coreEntityType: TLM_CET.DELETED, handler: this.handleMemberDeleted }
+    ])
+  }
+
+  handleMemberModified = data => {
+    const { props } = this
+    if (Number(props.userToEditId) !== data.fields.user.user_id) return
+    this.setState(prev => ({
+      workspaceList: prev.workspaceList.map(space =>
```

Comment:
Since you use space here, I think you can rename `workspaceList` into `spaceList`

Reply:
workspaceList is used everywhere in the frontend, I'd suggest to keep it like this for now so the same thing has the same name everywhere.

I opened an issue: https://github.com/tracim/tracim/issues/3734